EKF/gps_yaw_fusion.cpp

```diff
@@ -273,6 +259,7 @@ void Ekf::fuseGpsAntYaw()
 
 	// only apply covariance and state corrections if healthy
 	if (healthy) {
+		_time_last_gps_yaw_fuse = _time_last_imu;
```

Comment:
@kamilritz @priseborough Note that we usually have the `_time_last_xxx_fuse = _time_last_imu` line just after the innovation gate. This means that if not `healthy`, the fusion isn't performed but the timeout will never trigger. This is quite unlikely, but we might want to change that.